docs/templates/getting-started/faq.md

```diff
@@ -410,7 +411,7 @@ The VGG16 model is also the basis for several Keras example scripts:
 
 ### How can I use HDF5 inputs with Keras?
 
-You can use the `HDF5Matrix` class from `keras.utils.io_utils`. See [the documentation](/io_utils/#HDF5Matrix) for details.
+You can use the `HDF5Matrix` class from `keras.utils.io_utils`. See [the HDF5Matrix documentation](/io_utils/#HDF5Matrix) for details.
```

Comment:
"link here" or "the documentation" won't provide any semantic meaning to those with screen readers
https://www.sitepoint.com/15-rules-making-accessible-links/